src/structs/selections.py

```diff
@@ -5,18 +5,31 @@
 from cropped images.
 """
 from dataclasses import dataclass
-from typing import Any, Dict, Tuple
+from typing import Tuple, Optional
 
 
 @dataclass(slots=True)
 class ImageCropSelection:
     """Data structure containing metadata information of the selected
-    cropped images."""
+    cropped images.
+
+    Attributes:
+    -----------
+    img_id : int
+        unique number id that represents the cropped image
+
+    file_name : str
+        Name of the file where the cropped image came from.
+
+    crop_position : Tuple[int, int, int, int]
```

Comment:
please add `classification` to the class attributes docs

Reply:
I've added the documentation. I think it makes sense to let people know that there are private attributes associated with the `ImageCropSelection` data structure.